### PR TITLE
Anomaly events no longer crash and burn

### PR DESCRIPTION
**Pull request by Shuffl3**
Updates the_station_areas list so the events subsystem can hand out event targets again.

This has the nice side effect of not crashing the MC whenever events needing targets get triggered

fix: Anomalies are back bb

### CODE REVIEW

File: code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -2078,32 +2078,108 @@ var/list/teleportlocs = list()
...
+	/area/shuttle/ftl/cargo/qm,
+	/area/shuttle/ftl/cargo/mining,
+	/area/shuttle/ftl/turret_protected,
+	/area/shuttle/ftl/turret_protected/ai_upload,
+	/area/shuttle/ftl/turret_protected/ai,
+	/area/shuttle/ftl/telecomms/computer,
+	/area/shuttle/ftl/telecomms/server,
+	/area/shuttle/ftl/atmos,
+	/area/shuttle/ftl/atmos/equipment,
+	/area/shuttle/ftl/storage/tech,
```
Comment (monster860) [inline code/game/area/Space_Station_13_areas.dm]: There's no reason to include all the subtypes, just include all the top level types (except space)

Comment (Shuffl3) [inline code/game/area/Space_Station_13_areas.dm]: Neat

File: code/controllers/subsystem/events.dm [hunk 2]
```diff
@@ -102,18 +102,23 @@ var/datum/subsystem/events/SSevent
...
+	/area/shuttle/ftl/bridge,
+	/area/shuttle/ftl/crew_quarters/sleep,
+	/area/shuttle/ftl/engine/engine_smes,
+	/area/shuttle/ftl/engine/engineering,
+	/area/shuttle/ftl/munitions/cannon,
+	/area/shuttle/ftl/security/armory,
+	/area/shuttle/ftl/security/nuke_storage,
+	/area/shuttle/ftl/turret_protected,
+	/area/shuttle/ftl/turret_protected/ai_upload,
+	/area/shuttle/ftl/turret_protected/ai,
```
Comment (monster860) [inline code/controllers/subsystem/events.dm]: Include the escape shuttle in this list.

Comment (Shuffl3) [inline code/controllers/subsystem/events.dm]: Done

**Pull request merged**